authentication/src/main/AndroidManifest.xml

```diff
@@ -17,4 +17,39 @@
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <queries>
+        <package android:name="com.ubercab" />
+        <package android:name="com.ubercab.presidio.exo" />
```

Comment:
Do we want all 3rd party apps to work with dev builds?

Reply:
We supported these with the old sdk so I carried them over (maybe for dev purpose?). I'm ok with just restricting them to prod and beta builds.